app/src/main/java/com/codingbuffalo/aerialdream/ExoPlayerView.java

```diff
@@ -25,7 +25,7 @@
 import com.google.android.exoplayer2.upstream.cache.SimpleCache;
 import com.google.android.exoplayer2.video.VideoListener;
 
-public class ExoPlayerView extends TextureView implements MediaController.MediaPlayerControl, VideoListener, Player.EventListener {
+public class ExoPlayerView extends SurfaceView implements MediaController.MediaPlayerControl, VideoListener, Player.EventListener {
```

Comment:
Why the change to a surface view?

Reply:
I don't remember the exact error (since I've made this change quite a while ago), l but 4K playback wasn't working for me until I changed it to SurfaceView.

Something with hardware acceleration not working with TextureView, as far as I remember.

Reply:
Ok, thanks. SurfaceViews are (or were?) generally more efficient, but do not support transformations like transparency which your patch removes anyway.

Ok I'm merging this and will publish an update soon.